adam/tests/integration_tests/batch_propagation_test.py

```diff
@@ -0,0 +1,121 @@
+from adam import Service
+from adam import PropagationParams
+from adam import OpmParams
+from adam import ConfigManager
+from adam import BatchPropagation
+from adam import BatchPropagations
+
+import unittest
+import datetime
+import os
+
+
+class BatchPropagationTest(unittest.TestCase):
+
+    def setUp(self):
+        config = ConfigManager(os.getcwd() + '/test_config.json').get_config()
+        self.service = Service(config)
+        self.assertTrue(self.service.setup())
+        self.working_project = self.service.new_working_project()
+        self.assertIsNotNone(self.working_project)
+
+    def tearDown(self):
+        self.service.teardown()
+
+    def new_batch_propagation(self):
+        now = datetime.datetime.utcnow()
+        later = now + datetime.timedelta(10 * 365)
+
+        propagation_params = PropagationParams({
+            'start_time': now.isoformat() + 'Z',
+            'end_time': later.isoformat() + 'Z',
+            'step_size': 60 * 60,  # 1 hour.
+            'description': 'Created by test at ' + str(now) + 'Z'
+        })
+
+        state_vec = [130347560.13690618,
+                     -74407287.6018632,
+                     -35247598.541470632,
+                     23.935241263310683,
+                     27.146279819258538,
+                     10.346605942591514]
+        opm_params = OpmParams({
+            'epoch': now.isoformat() + 'Z',
+            'state_vector': state_vec,
+            # 'keplerian_elements': {
```

Comment:
Do you want these commented out?

Reply:
These yes, added comments - basically for ease of testing different kinds of propagations.